src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -245,86 +267,7 @@ void Data::ThermalCluster::setGroup(Data::ClusterName newgrp)
     }
     pGroup = newgrp;
     newgrp.toLower();
```

Comment:
This would probably find a better place inside the `stringToGroup` method since it seems to be used only for this search purpose.